vendor/bat-native-ledger/src/bat/ledger/internal/bat_state.cc

```diff
@@ -8,39 +8,40 @@
 
 #include "bat/ledger/internal/bat_state.h"
 #include "bat/ledger/internal/ledger_impl.h"
-#include "bat/ledger/internal/rapidjson_bat_helper.h"
+#include "bat/ledger/internal/state/client_state.h"
 
 namespace braveledger_bat_state {
 
 BatState::BatState(bat_ledger::LedgerImpl* ledger) :
       ledger_(ledger),
-      state_(new braveledger_bat_helper::CLIENT_STATE_ST()) {
+      state_(new ledger::ClientProperties()) {
```

Comment:
Just checking - why not have a member of `ledger::ClientProperties state_` to avoid dereferencing everywhere? Is it to avoid having ClientProperties included in `bat_state.h`? (If so, then np)

Reply:
Discussed and legacy code which is not part of this ticket